TTHAnalysis/macros/submit.py

```diff
@@ -0,0 +1,230 @@
+import os
+
+## this script creates a sub-directory in the tree path for a given module
+## and runs the friend tree producer for this module
+
+batch = True
+queue = "batch" #"batch" for Oviedo, "8nh" or others for lxplus
+path  = "/mnt_pool/fanae105/user/sscruz/TTH/samples"
```

Comment:
If we add it to this main repo, we should clean it up quite heavily, in order not to have (too many) hardcoded paths and such

@sscruz @nachosandres